TF migration docs - round 3 (#2038)
saucelabs/sauce-docs: docs/test-fairy/acct-mgmt/sso/sso-intro.md

---

> id: sso-intro

Author: Renamed file from "sso" to "sso-intro" because there's another file called "sso.md"

title: SSO
sidebar_label: SSO
---

import useBaseUrl from '@docusaurus/useBaseUrl';
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

Single Sign-on enables you to manage users and testers outside of TestFairy.
TestFairy supports SAML and is fully compatible with Okta, OneLogin, Ping, Oracle, IBM, and Azure ADFS.

> When SSO is configured into your account, the login page is replaced with a simple login with an SSO button.

Author: ```suggestion
When SSO is configured into your account, the login page is replaced with the login with an SSO button.
```

> There are several SAML / SSO providers that you can create integrations:

Author: ```suggestion
You can create integrations with several SAML/SSO providers:
```
